trunk/src/modules/stc.F90

```diff
@@ -342,31 +342,31 @@ subroutine stc_fwd_herm(Ni,Nb, Aii,Abi,Aib,Abb,Bi,Bb)
    VTYPE  , intent(inout) :: Aii(Ni,Ni),Abi(Nb,Ni),Aib(Ni,Nb),Abb(Nb,Nb), &
                              Bi(Ni,NR_RHS),Bb(Nb,NR_RHS)
 !
-!..workspace for Hermitian bubble matrix in packed format
+!..workspace for Hermitian bubble matrix in RFP format
    VTYPE, dimension(Nb*(Nb+1)/2) :: AP
 !
-   integer  :: i,j,k,nk,k1,k2
+   integer  :: i,j,k,k1,k2
```

Comment:
```suggestion
```

Reply:
?

Reply:
These integers were only used in the routine for converting to packed format, as far as I can see.